.eslintrc.yml

```diff
@@ -18,11 +18,14 @@ rules:
   no-unneeded-ternary: error
   no-useless-return: error
   no-var: error
+  object-shorthand: error
   one-var:
   - error
   - never
   prefer-arrow-callback: error
   prefer-const: error
+  prefer-destructuring: error
```

Comment:
Probably turn this one off as there are already 10 exceptions?

Reply:
I’ll change it to [stop looking at array accesses](https://eslint.org/docs/rules/prefer-destructuring#options).